src/main/java/meditracker/command/SaveCommand.java

```diff
@@ -46,33 +46,56 @@ public SaveCommand(String arguments)
     }
 
     /**
-     * Saves MediTracker information as JSON at its default save location specified in MediTrackerConfig.
+     * Saves MediTracker information to the default save location specified in `MediTrackerConfig`.
+     * Includes the Medication data (in JSON) and DailyMedication data (in txt).
      */
-    private void saveJsonToDefaultLocation() {
-        boolean isSaveSuccessful = FileReaderWriter.saveMediTrackerData(null);
+    private void saveToDefaultLocation() {
+        boolean isSaveSuccessful = FileReaderWriter.saveMedicationData(null);
         if (!isSaveSuccessful) {
-            System.out.println("An error occurred while saving. Data is not saved.");
+            System.out.println("An error occurred while saving Medication data. Data is not saved.");
+            return;
```

Comment:
Since both saving process should remain independent (i.e. should attempt saveDailyMedicationData), even if saveMedicationData fails, code should not return here.

Reply:
resolved: e8ab6759f4e39b486aa4286098edb631b93bc509